pkg/controller/cassandra/sts.go

```diff
@@ -45,7 +45,7 @@ spec:
             command:
             - /bin/bash
             - -c
-            - "seeds=$(for i in $(ls /mydata/*.yaml); do echo $(basename $i .yaml); done) &&  for seed in $(echo $seeds); do if [[ $(nodetool status | grep $seed |awk '{print $1}') != 'UN' ]]; then exit -1; fi; done"
+            - "if [[ $(nodetool status | grep ${POD_IP} |awk '{print $1}') != 'UN' ]]; then exit -1; fi;"
```

Comment:
Maybe this command could be simplified command [[ returns already 0 or 1. And kubelet just check of status code is 0

Reply:
I prefer explicitly specify when it fails. It's more readable this way. Also, I tried to use expression inside [[]] and it didn't work. I guess I missed something. But my main point if you don't explicitly specifies the exit condition then it's not clear.

Reply:
ok